### PR TITLE
konvoy: document that creating an override is required for preprov clusters with an HTTP proxy

### PR DESCRIPTION
**Pull request by dkoshkin**
## Jira Ticket

Fixeshttps://jira.d2iq.com/browse/D2IQ-89698.
Documentation for https://jira.d2iq.com/browse/COPS-7214.
I created a separate task https://jira.d2iq.com/browse/D2IQ-89695 for code changes.

## Description of changes being made

Added a Note about users needing to create an override to hold their HTTP proxy configuration. Also fixed the `proxy-or-files/index.md` and made them all consistent across releases.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4450.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 1]
```diff
@@ -175,6 +175,8 @@ If you require HTTP proxy configurations, you can apply them during the `create`
 | HTTPS proxy for worker machines          | `--worker-https-proxy string`        |
 | No Proxy list for worker machines        | `--worker-no-proxy strings`          |
 
+<p class="message--note"><strong>NOTE: </strong>You must also add the same configuration as an a href="../create-secrets-and-overrides#create_overrides">override</a>. See href="../../../image-builder/override-files/create-custom-or-files/proxy-or-files">this documentation</a> [here].</p>
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-cluster/index.md:178]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>You must also add the same configuration as an <a href="../create-secrets-and-overrides#create_overrides">override</a>. For more information, refer to <a href="../../../image-builder/override-files/create-custom-or-files/proxy-or-files">this documentation</a>.</p>
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 2]
```diff
@@ -206,6 +206,8 @@ If you require HTTP proxy configurations, you can apply them during the `create`
 | HTTPS proxy for worker machines          | `--worker-https-proxy string`        |
 | No Proxy list for worker machines        | `--worker-no-proxy strings`          |
 
+<p class="message--note"><strong>NOTE: </strong>You must also add the same configuration as an a href="../create-secrets-and-overrides#create_overrides">override</a>. See href="../../../image-builder/override-files/create-custom-or-files/proxy-or-files">this documentation</a> [here].</p>
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md:209]: Links are broken 
```suggestion
<p class="message--note"><strong>NOTE: </strong>You must also add the same configuration as an <a href="../create-secrets-and-overrides#create_overrides">override</a>. For more information, refer to <a href="../../../image-builder/override-files/create-custom-or-files/proxy-or-files">this documentation</a>.</p>
```

File: pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 3]
```diff
@@ -206,6 +206,8 @@ If you require HTTP proxy configurations, you can apply them during the `create`
 | HTTPS proxy for worker machines          | `--worker-https-proxy string`        |
 | No Proxy list for worker machines        | `--worker-no-proxy strings`          |
 
+<p class="message--note"><strong>NOTE: </strong>You must also add the same configuration as an a href="../create-secrets-and-overrides#create_overrides">override</a>. See href="../../../image-builder/override-files/create-custom-or-files/proxy-or-files">this documentation</a> [here].</p>
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-cluster/index.md:209]: Links are broken 
```suggestion
<p class="message--note"><strong>NOTE: </strong>You must also add the same configuration as an <a href="../create-secrets-and-overrides#create_overrides">override</a>. For more information, refer to <a href="../../../image-builder/override-files/create-custom-or-files/proxy-or-files">this documentation</a>.</p>
```

**Review by gvasquezvargas — COMMENTED** [review]
Some links are broken, posted suggestions.

**Review by gvasquezvargas — APPROVED** [review]

**Pull request merged**